### DISCUSSION TITLE
How to use mutiple FE framework in one project

### DISCUSSION THREAD

**Discussion by nhattran998** [vikejs/vike#1762] [Discussion] [answered]
Use cases here: I usually use React for the SPA page, but with the marketing page, I prefer SolidJs to optimize performance & SEO points. 

Why don't you use Solid Js for the whole page? Because the ecosystem around React is incredible, especially some text editors, animation, or motion libs only have in React(well maintained & popular).

So, do you know how I can archive this with Vike? 

Thanks for reading <3

**Comment by brillout** [comment, marked as answer]
Yes, you can do that, for example [by using both vike-react and vike-solid](https://vike.dev/config#powerful). (If you use TypeScript then make sure to use `ConfigCore & ConfigReact` and `ConfigCore & ConfigSolid` instead of `Config`. These types aren't exported yet but PR welcome.)
